modules/nw-metallb-operator-setting-runtimeclass.adoc

```diff
@@ -47,12 +47,12 @@ spec:
     runtimeClassName: myclass
     annotations: <1>
       controller: demo
-  speakerConfig: 
+  speakerConfig:
     runtimeClassName: myclass
     annotations: <1>
       speaker: demo
 ----
-<1> This example uses `annotations` to add metadata such as build release information or GitHub pull request information. You can populate annotations with characters not permitted in labels. However, you cannot use annotations to identify or select objects. 
+<1> This example uses `annotations` to add metadata such as build release information or GitHub pull request information. You can populate annotations with characters not permitted in labels. However, you cannot use annotations to identify or select objects.
```

Comment:
This example > The previous example is more direct for accessibility requirements.

Reply:
not permitted > that are no permitted.